### PR TITLE
Support package json config

### PR DESCRIPTION
**Pull request by digitalsadhu**

### CODE REVIEW

File: src/plugin.js [hunk 1]
```diff
@@ -10,18 +10,29 @@ const notUrl = (url) => url.substr(0, 4) !== 'http';
...
+        return JSON.parse(contents);
     } catch (err) {
-        return [];
+        return {};
     }
 }
 
+async function readEikJSONMaps(eikJSONPath, pkgJSONPath) {
+    const eikJSON = await readJSONFile(eikJSONPath);
+    const pkgJSON = await readJSONFile(pkgJSONPath);
```
Comment (trygve-lie) [inline src/plugin.js:24]: We're using promises here to be non blocking but there is no real benefit to this due to each step being done in a sync order.

If we make `readJSONFile` into returning a promise which resolve with a object or rejects, these two lines can be run in an awaited `Promise.all()` and both files will be read in parallel. Which should improve speed.

File: README.md [hunk 2]
```diff
@@ -82,15 +82,60 @@ module.exports = (ctx) => ({
...
+
+```js
+module.exports = (ctx) => ({
+    plugins: [
+        require('@eik/postcss-import-map')({ path: '/path/to/eik.json' }),
+        require('postcss-import')({
+            // It needs to be added here as well to ensure everything is mapped
+            plugins: [
+                require('@eik/postcss-import-map')({
+                    path: '/path/to/eik.json',
```
Comment (stipsan) [inline README.md:100]: @digitalsadhu why are the snippets here using absolute paths, when the API options docs later on use relative paths: `cwd/eik.json`?

**Review by stipsan — APPROVED** [review]
LGTM

File: README.md (hunk 2 above)
Comment (digitalsadhu) [reply README.md:100]: Don't really have a good answer or reason.

**Comment by trygve-lie** [comment]
I'm merging this into the `next` branch to do a bit of a rework on this module.

**Pull request merged**